examples/prometheus/README.md

```diff
@@ -0,0 +1,137 @@
+# Monitoring STUNner with Prometheus and Grafana
+
+This tutorial demonstrates monitoring of a STUNner instance with [Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/).
+
+In this demo you will learn how to:
+* monitor STUNner with [Prometheus](https://prometheus.io/)
+* how to visualize STUNner metrics using [Grafana](https://grafana.com/)
+
+## Installation
+
+### Prerequisites
+
+* The tutorial assumes a fresh STUNner installation in a namespace called `stunner`; see the [STUNner installation and configuration guide](/doc/INSTALL.md).
+* You need a web browser to run this tutorial.
+
+### Enable STUNner metrics endpoint
+
+After installing STUNner, enable the STUNner metrics endpoint:
+```console
+kubectl -n stunner patch gatewayconfigs.stunner.l7mp.io stunner-gatewayconfig --patch '{"spec": {"metricsEndpoint": "http://0.0.0.0:8080/metrics" }}' --type=merge
+```
+
+Patch STUNner deployment to enable the monitoring port (required by the pod monitor):
+```console
+kubectl -n stunner patch deployment stunner --patch '{"spec":{"template":{"spec":{"containers":[{"name":"stunnerd","ports":[{"name": "web","containerPort": 8080 }]}]}}}}'
+```
+
+### Deploy Prometheus and Grafana
+
+Deploy Prometheus with `helm`:
+```console
+helm install prometheus stunner/stunner-prometheus
```

Comment:
What is stunner specific in this prometheus install? A couple of words could be useful to the reader why we do this.